wger/exercises/tests/test_categories.py

```diff
@@ -81,49 +81,49 @@ class AddExerciseCategoryTestCase(WorkoutManagerAddTestCase):
     url = 'exercise:category:add'
     data = {'name': 'A new category'}
 
-
-class ExerciseCategoryCacheTestCase(WorkoutManagerTestCase):
-    """
-    Cache test case
-    """
-
-    def test_overview_cache_update(self):
-        """
-        Test that the template cache for the overview is correctly reseted when
-        performing certain operations
-        """
-
-        self.client.get(reverse('exercise:exercise:overview'))
-        self.client.get(reverse('exercise:exercise:view', kwargs={'id': 2}))
-
-        old_exercise_overview = cache.get(
-            get_template_cache_name('exercise-overview', 2))
-        old_exercise_overview_mobile = cache.get(
-            get_template_cache_name('exercise-overview-mobile', 2))
-
-        category = ExerciseCategory.objects.get(pk=2)
-        category.name = 'Cool category'
-        category.save()
-
-        self.assertFalse(
-            cache.get(get_template_cache_name('exercise-overview', 2)))
-        self.assertFalse(
-            cache.get(get_template_cache_name('exercise-overview-mobile', 2)))
-
-        self.client.get(reverse('exercise:exercise:overview'))
-        self.client.get(reverse('exercise:muscle:overview'))
-        self.client.get(reverse('exercise:exercise:view', kwargs={'id': 2}))
-
-        new_exercise_overview = cache.get(
-            get_template_cache_name('exercise-overview', 2))
-        new_exercise_overview_mobile = cache.get(
-            get_template_cache_name('exercise-overview-mobile', 2))
-
-        if not self.is_mobile:
-            self.assertNotEqual(old_exercise_overview, new_exercise_overview)
-        else:
-            self.assertNotEqual(old_exercise_overview_mobile,
-                                new_exercise_overview_mobile)
+# Disabled cache in exercise view for filtering purposes so all cache tests are disabled also
```

Comment:
I don't think to comment out failing test cases is the right thing to do.

Reply:
I dropped the caching functionality from the exercises as is when you filter the caching feature returns a cached copy of the previous exercises. So the test are not functional @collin5

Reply:
@sir3n-sn Could you maybe just modify the caching logic to un-cache the page once the language is different from the previous. Coz we can't just get rid of the caching functionality.

Reply:
On it